shared/AppInsightsCommon/src/Util.ts

```diff
@@ -335,24 +341,85 @@ export class Util {
         return CoreUtils._canUseCookies && Util.document && Util.document.cookie;
     }
 
+    public static disallowsSameSiteNone(userAgent:string) {
+        if (!_isString(userAgent)) {
+            return false;
+        }
+    
+        // Cover all iOS based browsers here. This includes:
+        // - Safari on iOS 12 for iPhone, iPod Touch, iPad
+        // - WkWebview on iOS 12 for iPhone, iPod Touch, iPad
+        // - Chrome on iOS 12 for iPhone, iPod Touch, iPad
+        // All of which are broken by SameSite=None, because they use the iOS networking stack
+        if (userAgent.indexOf("CPU iPhone OS 12") !== -1 || userAgent.indexOf("iPad; CPU OS 12") !== -1) {
+            return true;
+        }
+     
+        // Cover Mac OS X based browsers that use the Mac OS networking stack. This includes:
+        // - Safari on Mac OS X
+        // This does not include:
+        // - Internal browser on Mac OS X
+        // - Chrome on Mac OS X
+        // - Chromium on Mac OS X
+        // Because they do not use the Mac OS networking stack.
+        if (userAgent.indexOf("Macintosh; Intel Mac OS X 10_14") !== -1 && userAgent.indexOf("Version/") !== -1 && userAgent.indexOf("Safari") !== -1) {
+            return true;
+        }
+     
+        // Cover Mac OS X internal browsers that use the Mac OS networking stack. This includes:
+        // - Internal browser on Mac OS X
+        // This does not include:
+        // - Safari on Mac OS X
+        // - Chrome on Mac OS X
+        // - Chromium on Mac OS X
+        // Because they do not use the Mac OS networking stack.
+        if (userAgent.indexOf("Macintosh; Intel Mac OS X 10_14") !== -1 && _endsWith(userAgent, "AppleWebKit/605.1.15 (KHTML, like Gecko)")) {
+            return true;
+        }
+     
+        // Cover Chrome 50-69, because some versions are broken by SameSite=None, and none in this range require it.
+        // Note: this covers some pre-Chromium Edge versions, but pre-Chromim Edge does not require SameSite=None, so this is fine.
+        // Note: this regex applies to Windows, Mac OS X, and Linux, deliberately.
+        if (userAgent.indexOf("Chrome/5") !== -1 || userAgent.indexOf("Chrome/6") !== -1) {
+            return true;
+        }
+     
+        // Unreal Engine runs Chromium 59, but does not advertise as Chrome until 4.23. Treat versions of Unreal
+        // that don't specify their Chrome version as lacking support for SameSite=None.
```

Comment:
I cannot see this in https://www.chromium.org/updates/same-site/incompatible-clients are you using some other reference for this specific user agents?